_includes/_pages/blockly.html

```diff
@@ -0,0 +1,17 @@
+{% assign t = site.data.translations.[page.lang].blockly %}
+<div class="page-header">
+  <h1>Blockly für senseBox</h1>
+</div>
+{{ t.description }}
+<div class="page-header">
+  <h1>Installieren</h1>
+</div>
+{{ t.description_installieren }}
+
+<p>&nbsp;</p>
+
+<p style="text-align:center"><a class="btn" href="https://sensebox.de/blockly" target="_blank">{{ t.link }}</a></p>
+
+<p>&nbsp;</p>
+
+<p style="text-align:center"><a href="https://sensebox.de/blockly" target="_blank"><img class="responsive-img" src="{{ site.baseurl | append: '/images/blog_images/ardublockly1.gif'}}" alt="Blockly"></a></p>
```

Comment:
@mariopesch kannst du hier noch ein aktuelles gif hinzufügen. In der Variante die online ist, gibt es die Kategorie `senseBox Grundlagen` nicht mehr.